<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView_General"/>
      <widget class="QTableView" name="tableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="action_creat"/>
    <addaction name="action_open"/>
    <addaction name="action_saveAs"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Правка</string>
    </property>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_delete"/>
    <addaction name="action_selectAll"/>
    <addaction name="separator"/>
    <addaction name="action_addAtr"/>
    <addaction name="separator"/>
    <addaction name="action_find"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справ&amp;ка</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_aboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Вид</string>
    </property>
    <addaction name="action_viewTree"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_viewTree"/>
   <addaction name="action_open"/>
   <addaction name="action_addAtr"/>
   <addaction name="action_find"/>
   <addaction name="action_delete"/>
   <addaction name="action_exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="text">
    <string>Открыть. . .</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="text">
    <string>Сохранить как. . .</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_creat">
   <property name="text">
    <string>Создать новый файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icon/exit.png</normaloff>:/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_find">
   <property name="text">
    <string>Поиск</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_selectAll">
   <property name="text">
    <string>Выделить все</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>Информация о программе</string>
   </property>
  </action>
  <action name="action_aboutQt">
   <property name="text">
    <string>Информация о программе Qt Creator</string>
   </property>
  </action>
  <action name="action_viewTree">
   <property name="text">
    <string>&amp;Показать модель дерева</string>
   </property>
  </action>
  <action name="action_addAtr">
   <property name="text">
    <string>Добавить новый атрибут</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
